Source/gs/GSHandler.cpp

```diff
@@ -552,7 +552,7 @@ void CGSHandler::WriteRegisterImpl(uint8 nRegister, uint64 nData)
 
 void CGSHandler::FeedImageDataImpl(const void* pData, uint32 nLength)
 {
-	boost::scoped_array<const uint8> dataPtr(reinterpret_cast<const uint8*>(pData));
+	std::unique_ptr<const uint8> dataPtr(reinterpret_cast<const uint8*>(pData));
```

Comment:
Not sure it's going to work on all compilers, but using a `std::unique_ptr<const uint8[]>` would seem like a better idea since we're using `new []` in `FeedImageData` and we want to make sure `delete []` is called.

Reply:
👍  just reading [cppreference ](https://en.cppreference.com/w/cpp/memory/default_delete) right now, it seems to be a c++17 feature, which means we'd need to move standard up (but as you mentioned, pending compiler support, cough apple cough)

Reply:
Oh, I didn't notice the C++17 part... Well, there's another way to handle this case. I'll make the change and push it in master.